src/WaveClip.cpp

```diff
@@ -1772,8 +1700,8 @@ void WaveClip::Resample(int rate, ProgressDialog *progress)
 // be exactly equal due to rounding errors.
 bool WaveClip::SharesBoundaryWithNextClip(const WaveClip* next) const
```

Comment:
The comment preceding this function also needs update to mention new method names